GameData/RealismOverhaul/Engine_Configs/AJ10_Adv_Config.cfg

```diff
@@ -63,9 +63,13 @@
 				key = 0 311
 				key = 1 100
 			}
-			ullage = True
-			pressureFed = True
-			ignitions = 0
+
+			%ullage = True
```

Comment:
And all these should be removed once again.

Reply:
I thought this stuff had it fixed after your conversation?

Reply:
I just saw that you merged the current master updates into this PR: https://github.com/KSP-RO/RealismOverhaul/pull/1978/commits/4c275c18e3efbd315a601771c036f40395a0ec1b#diff-4cb441499a2706b72f87706c6200f9c9L67

Reply:
I fixed it by force pushing the state before the merge commit, probably the cleanest solution, everything should be good again.